perception/object_recognition/tracking/multi_object_tracker/CMakeLists.txt

```diff
@@ -38,20 +33,28 @@ add_executable(multi_object_tracker_node
   src/data_association/successive_shortest_path/successive_shortest_path.cpp
 )
 
-add_dependencies(multi_object_tracker_node
-  ${${PROJECT_NAME}_EXPORTED_TARGETS}
-  ${catkin_EXPORTED_TARGETS}
+set(MULTI_OBJECT_TRACKER_HEADERS
+  src/data_association/successive_shortest_path/successive_shortest_path.hpp
+  include/multi_object_tracker/data_association/data_association.hpp
+  include/multi_object_tracker/tracker/tracker.hpp
+  include/multi_object_tracker/tracker/model/tracker_base.hpp
+  include/multi_object_tracker/tracker/model/bicycle_tracker.hpp
+  include/multi_object_tracker/tracker/model/pedestrian_tracker.hpp
+  include/multi_object_tracker/tracker/model/vehicle_tracker.hpp
+  include/multi_object_tracker/utils/utils.hpp
+  include/multi_object_tracker/multi_object_tracker_core.hpp
 )
```

Comment:
ament_cmake_auto should add include file automatically, so you shouldn't have to set this.

Reply:
I was told in a different PR that this is better practice: https://github.com/tier4/Pilot.Auto/pull/18#discussion_r504575682. I figured it applied here too, maybe I'm wrong.